tools/test-backend

```diff
@@ -54,12 +54,6 @@ target_fully_covered = {path for target in [
 ] for path in glob.glob(target)}
 
 not_yet_fully_covered = {path for target in [
-    # Analytics fixtures library is used to generate test fixtures;
```

Comment:
Removed these files because of a message from the CI/CD pipeline: https://app.circleci.com/pipelines/github/zulip/zulip/11105/workflows/46bd2451-a473-4bd2-b881-f09fb3ac7b55/jobs/63670

Reply:
Strange; I don't see how the coverage for those files would have changed; `test-backend --coverage` for me shows them as still incomplete.

Reply:
I'll revert these changes and see if I can figure out what is going on in the CI pipeline. 

Not sure if it will fix the coverage issue, but I think I need to rewrite the test I added to avoid calling `populate_db`. Some tests in `test_sub.py` fail when they run after this test. 

```python
class TestDevelopMessages(ZulipTestCase):

    def setUp(self) -> None:
        super().setUp()
        out = io.StringIO()
        with self.assertLogs():
            call_command('populate_db', stdout=out)

    def tearDown(self) -> None:
        clear_database()
        super().setUp()
```

Reply:
Yeah, it's not a good idea to run `populate_db` in a test (if nothing else, it's slow!); what I'd do is extract the logic to e.g. `zerver/lib/bulk_create.py` and then unit-test that.